manifests/t/Twingate/Client/1.0.33.24402/Twingate.Client.locale.en-US.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Twingate.Client
+PackageVersion: 1.0.33.24402
+PackageLocale: en-US
+Publisher: Twingate Inc.
+PublisherUrl: https://www.twingate.com/
+PublisherSupportUrl: https://help.twingate.com/hc/en-us

```

Comment:
```suggestion
# PublisherSupportUrl: 
```
The `/hc` in the path suggests that this is probably a ZenDesk Help Center, and unfortunately, our automated connection attempts are blocked.